src/components/Composer.vue

```diff
@@ -1097,9 +1209,23 @@ export default {
 
 .composer-fields {
 	display: flex;
-	align-items: center;
+	// align-items: flex-start;
```

Comment:
this blocks the draft to be saved because its not positioned right. Having it like this: align-items: center; causes no problems

Reply:
I don't quite understand what this is about. I removed the `align-item:center` only because I wanted to position the field headers on top. in theory, they can do `align-self:flex-start` and then, if there are a lot of recipients, then it seems to me that it is more correct. 
This screen is with .composer-fields `align-item:center` property.
![image](https://user-images.githubusercontent.com/3595562/167143433-365be41a-8564-4504-a6b8-ff563cb8edee.png)
Now I will make a commit with the ill-fated comma, which ruined the whole holiday ... =)

Reply:
oh damn :( sorry, i am totally fine to push stuff on your pr, so if you want to take over some parts of the comments i make, please let me know in the future, specially if its small typo like this one.

Reply:
> I don't quite understand what this is about. I removed the `align-item:center` only because I wanted to position the field headers on top. in theory, they can do `align-self:flex-start` and then, if there are a lot of recipients, then it seems to me that it is more correct. This screen is with .composer-fields `align-item:center` property. ![image]

You might be right, i didnt test it with a lot of recipients, let me have a second look